### PR TITLE
[php] Add README.md

### PR DESCRIPTION
**Pull request by cw-ozaki**
Added [README.md](https://github.com/chatwork/charts/blob/67f2a42a70e7e7dc3fe1e867e6484c8af9927589/php/README.md). And fixed consistency and complexity.

### CODE REVIEW

File: php/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+| --- | --- | --- |
+|  `nginx.image.repository` | The image repository to pull from | `nginx` |
+|  `nginx.image.tag` | The image tag to pull | `1.15.1-alpine` |
+|  `nginx.image.pullPolicy` | Image pull policy | `IfNotPresent` |
+|  `nginx.port` | Default public port of NGINX | `80` |
+|  `nginx.livenessProbe` | Overrides the default liveness probe | HTTP `/status` port `7777` |
+|  `nginx.readinessProbe` | Overrides the default readness probe | HTTP `/ping` port `7777` |
+|  `nginx.resources` | Overrides the default resource | `{}` |
+|  `nginx.extraEnv` | Additional environment variables | `{}` |
+|  `nginx.extraEnvFrom` | Additional envFrom | `{}` |
```
Comment (cw-sakamoto) [inline php/README.md]: Values is ```[]``` ?
and
- ```nginx.extraPorts``` as same.
- ```nginx.extraVolumes``` is seemed to be not here
- ```nginx.etraVolumeMounts``` is seemed to be correct```{}```(in README), but ```[]``` in values.

fpm parameters are likely to make the same mistake.

File: php/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+## Configuration
+
+The following table lists the configurable parameters of the PHP chart and their default values.
+
+|  Parameter | Description | Default |
+| --- | --- | --- |
+|  `sharedPath` | Path shared by busybox/nginx/fpm | `/var/www/html` |
+|  `replicaCount` | Number of pods to start with deployment | `1` |
+|  `strategy` | Update strategy of pod in deployment | `type`: `RollingUpdate` |
```
[the remark this answers is not in the document]
Comment (cw-ozaki) [reply php/README.md]: @cw-sakamoto 
I referred to this.
https://github.com/chatwork/charts/blob/67f2a42a70e7e7dc3fe1e867e6484c8af9927589/php/values.yaml#L11-L12
YES

File: php/values.yaml [hunk 3]
```diff
@@ -737,44 +918,27 @@ fpm:
...
-#       type: Opaque
-#       data:
-#         hoge: {{ .Values.extras.hoge }}
-
+  # Grant annotations to ConfigMap of `fpm.templates`, Secrets of `fpm.secrets`.
+  annotations: {}
+
+# `extra` is additional Raw Kubernetes Config as a string to be passed to the tpl function.
+extras:
+  templates: []
```
[the remark this answers is not in the document]
Comment (cw-ozaki) [reply php/values.yaml]: oops!

File: php/README.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+| --- | --- | --- |
+|  `fpm.image.repository` | The image repository to pull from | `php` |
+|  `fpm.image.tag` | The image tag to pull | `7.1-fpm-alpine` |
+|  `fpm.image.pullPolicy` | Image pull policy | `IfNotPresent` |
+|  `fpm.ini` | Override the default php.ini. Please specify the parameter name with Lower Camel Case |  |
+|  `fpm.livenessProbe` | Overrides the default liveness probe | `{}` |
+|  `fpm.readinessProbe` | Overrides the default readness probe | `{}` |
+|  `fpm.resources` | Overrides the default resource | `{}` |
+|  `fpm.extraEnv` | Additional environment variables | `{}` |
+|  `fpm.extraEnvFrom` | Additional envFrom | `[] |
```
Comment (ada-u) [inline php/README.md]: めっちゃ些細ですけどw、```が抜けてますね、`[]`。

**Comment by cw-sakamoto** [comment]
LGTM😇

**Pull request merged**